### PR TITLE
fixed concat error

### PR DESCRIPTION
**Pull request by JATaber**
This pull request fixed a concatenation error, some minor spelling, and the addition of the .gitignore

### CODE REVIEW

File: src/FormBuilder.js [hunk 1]
```diff
@@ -7,12 +7,12 @@ buildForm: function(formItems){
...
   //looping through all of the items in the array to create the form
   formItems.forEach((items)=>{
-    //verifying that ther is a type for each of the items if not an error will be thrown
+    //verifying that there is a type for each of the items if not an error will be thrown
     if(items.type){
       //checking the types
       switch (items.type){
         case 'select':
-          formInput+= '<select value='+items.value+'>'+items.value'\n';
+          formInput+= '<select value='+items.value+'>'+items.value+'\n';
```
Comment (JATaber) [inline src/FormBuilder.js]: This was the main error where you forgot the addition operator to concatenate the variable in the string

Comment (JATaber) [inline src/FormBuilder.js:15]: added an addition operator to properly concatenate the variable inside of the string

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.idea
+node_modules
```
Comment (JATaber) [inline .gitignore:2]: added in the .gitignore since it was excluded from the fork

File: src/FormBuilder.js [hunk 3]
```diff
@@ -51,7 +51,7 @@ buildForm: function(formItems){
     }
     //if the type is not included in the array an error is thrown
     else{
-      console.log("Oops! It looks like you didnt enter the correct information double check your array to make sure it follows the correct syntax.")
+      console.log("Oops! It looks like you didn't enter the correct information double check your array to make sure it follows the correct syntax.")
```
Comment (JATaber) [inline src/FormBuilder.js:54]: this was just a minor spelling error

File: src/FormBuilder.js [hunk 4]
```diff
@@ -7,12 +7,12 @@ buildForm: function(formItems){
   let formInput='';
   //looping through all of the items in the array to create the form
   formItems.forEach((items)=>{
-    //verifying that ther is a type for each of the items if not an error will be thrown
+    //verifying that there is a type for each of the items if not an error will be thrown
```
Comment (JATaber) [inline src/FormBuilder.js:10]: minor spelling error

File: .gitignore (hunk 2 above)
Comment (mpeck99) [reply .gitignore:2]: Thank you for that! I completely forgot about that.

File: src/FormBuilder.js (hunk 1 above)
Comment (mpeck99) [reply src/FormBuilder.js:15]: Thanks for finding that error I thought I caught them all.

File: src/FormBuilder.js (hunk 4 above)
Comment (mpeck99) [reply src/FormBuilder.js:10]: Thanks!

**Pull request merged**